keyboards/planck/keymaps/ariccb/keymap.c

```diff
@@ -0,0 +1,182 @@
+
+// Upper has a layout that's optimized for coding symbols, and has a few duplicates just to entertain my existing muscle memory. Some 
+// things apparently can't be un-taught.
+
+// CAPS has moved to the Fn layer, and a few additional shortcut modifiers like CTRL_ALT_UP and DOWN for adding additional cursors in VSCode.
+// Play/Pause has a prime spot on the base layer, and the Fn version skips to next track, with left and right on the base changing to Down and Up
+// on the Fn layer. Volume Up and Volume Down live on bottom left of Fn Layer.
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+/* MIT Layout (QWERTY layer)
+ *
+ * ,-----------------------------------------------------------------------.
+ * |FN,ESC|  q  |  w  |  e  |  r  |  t  |  y  |  u  |  i  |  o  |  p  |Bspc|
+ * |-----------------------------------------------------------------------|
+|HYPER,Tab|  a  |  s  |  d  |  f  |  g  |  h  |  j  |  k  |  l  |  ;  |  ' |
+ * |-----------------------------------------------------------------------|
+ * |shift |  z  |  x  |  c  |  v  |  b  |  n  |  m  |  ,  |  .  |  /  |  \ |
+ * |-----------------------------------------------------------------------|
+ * | Ctl  | Win | Alt |Enter|LOWER|   Space   |RAISE|Menu |Play | Lft |Rght|
+ * `-----------------------------------------------------------------------'
+ */
+[_QWERTY] = LAYOUT_planck_grid( /* QWERTY */
+    LT(_FN, KC_ESC),    KC_Q,     KC_W,    KC_E,   KC_R,  KC_T,   KC_Y,   KC_U,  KC_I,    KC_O,    KC_P,    KC_BSPC,
+    LT(KC_HYPR, KC_TAB),KC_A,     KC_S,    KC_D,   KC_F,  KC_G,   KC_H,   KC_J,  KC_K,    KC_L,    KC_SCLN, KC_QUOT, 
+    KC_LSFT,            KC_Z,     KC_X,    KC_C,   KC_V,  KC_B,   KC_N,   KC_M,  KC_COMM, KC_DOT,  KC_SLSH, KC_BSLS, 
+    KC_LCTL,            KC_LGUI,  KC_LALT, KC_ENT, LOWER, KC_SPC, KC_SPC, RAISE, KC_APP,  KC_MPLY, KC_LEFT, KC_RGHT
+ ),
+
+ /* MIT Layout (COLEMAK layer)
+ *
+ * ,-----------------------------------------------------------------------.
+ * |FN,ESC|  q  |  w  |  f  |  p  |  b  |  j  |  l  |  u  |  y  |  ;  |Bspc|
+ * |-----------------------------------------------------------------------|
+|HYPER,Tab|  a  |  r  |  s  |  t  |  g  |  h  |  n  |  e  |  i  |  o  |  ' |
+ * |-----------------------------------------------------------------------|
+ * |shift |  z  |  x  |  c  |  d  |  v  |  k  |  m  |  ,  |  .  |  /  |  \ |
+ * |-----------------------------------------------------------------------|
+ * | Ctl  | Win | Alt |Enter|LOWER|   Space   |RAISE|Menu |Play | Lft |Rght|
+ * `-----------------------------------------------------------------------'
+ */
+[_COLEMAK] = LAYOUT_planck_grid( /* COLEMAK */
+    LT(_FN, KC_ESC),    KC_Q,    KC_W,    KC_F,   KC_P,  KC_B,   KC_J,   KC_L,  KC_U,    KC_Y,    KC_SCLN, KC_BSPC,
+    LT(KC_HYPR, KC_TAB),KC_A,    KC_R,    KC_S,   KC_T,  KC_G,   KC_M,   KC_N,  KC_E,    KC_I,    KC_O,    KC_QUOT, 
```

Comment:
LT require a layer, not a keycode.

Reply:
oops, was intending on a Mod-Tap there, not a layer switch.